components/standings/commons/standings_parse_wiki.lua

```diff
@@ -0,0 +1,105 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Standings/Parse/Wiki
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Json = require('Module:Json')
+local Logic = require('Module:Logic')
+local Table = require('Module:Table')
+
+local OpponentLibrary = require('Module:OpponentLibraries')
+local Opponent = OpponentLibrary.Opponent
+
+local StandingsParseWiki = {}
+
+--[[
+{{FfaStandings|title=League Standings
+|bg=1-10=stay, 11-20=down
```

Comment:
not a fan of this format for the bg stuff
maybe json it?

Reply:
It's the same as FFA matches. We can look at changing both of them if you like, but I think i'll leave you in charge of that then :D

Reply:
not a blocker, just mentioning that i don't like it^^